### PR TITLE
Add example to "add user to channel using UPN"

### PR DESCRIPTION
**Pull request by devjha-ms**
Add example for "Add a member to a channel using user principal name"

### CODE REVIEW

**Review by shantanu1singh — APPROVED** [review]
LGTM

File: api-reference/beta/api/channel-post-members.md [hunk 1]
```diff
@@ -433,6 +433,51 @@ Content-length: 468
     "tenantId": "a18103d1-a6ef-4f66-ac64-e4ef42ea8681"
 }
 ```
+### Example 6: Add a user as a member to a channel using user principal name
```
Comment (JarbasHorst) [inline api-reference/beta/api/channel-post-members.md:436]: Hi @devjha-ms, according to the name of your branch "devjha-add-shared-channel-member-using-upn" should the title for this example be: "Add a user who is part of the same tenant as a member to a shared channel using user principal name"?

Comment (devjha-ms) [reply api-reference/beta/api/channel-post-members.md:437]: This example is common for both whether the member is from the same tenant or external tenant.

Update tile to reflect the same.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**